integration-tests/src/test/java/oracle/verrazzano/weblogic/kubernetes/ItVzMiiDomain.java

```diff
@@ -197,7 +217,202 @@ void testCreateVzMiiDomain() {
+   */
+  @Test
+  @DisplayName("Verrazzano mill domain with auxilary image")
+  void testVzMiiAuxilaryImage() {
+    String auxDomainUid = "aux-domain";
+    String miiAuxiliaryImage1Tag = "image1" + MII_BASIC_IMAGE_TAG;
+    String miiAuxiliaryImage1 = MII_AUXILIARY_IMAGE_NAME + ":" + miiAuxiliaryImage1Tag;
+    String miiAuxiliaryImage2Tag = "image2" + MII_BASIC_IMAGE_TAG;
+    String miiAuxiliaryImage2 = MII_AUXILIARY_IMAGE_NAME + ":" + miiAuxiliaryImage2Tag;
     
+    final int replicaCount = 2;
+    final String adminServerPodName = auxDomainUid + "-admin-server";
+    final String managedServerPrefix = auxDomainUid + "-managed-server";
+
+    String adminSecretName = "weblogic-credentials";
+    String encryptionSecretName = "encryptionsecret";
+
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    createSecretWithUsernamePassword(adminSecretName, auxDomainNamespace,
+        ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
+
+    // create encryption secret
+    logger.info("Create encryption secret");
+    createSecretWithUsernamePassword(encryptionSecretName, auxDomainNamespace,
+        "weblogicenc", "weblogicenc");
+
+    // build app
+    assertTrue(buildAppArchive(defaultAppParams()
+        .srcDirList(Collections.singletonList(MII_BASIC_APP_NAME))
+        .appName(MII_BASIC_APP_NAME)),
+        String.format("Failed to create app archive for %s", MII_BASIC_APP_NAME));
+
+    // image1 with model files for domain config, ds, app and wdt install files
+    List<String> archiveList = Collections.singletonList(ARCHIVE_DIR + "/" + MII_BASIC_APP_NAME + ".zip");
+
+    List<String> modelList = new ArrayList<>();
+    modelList.add(MODEL_DIR + "/" + MII_BASIC_WDT_MODEL_FILE);
+    modelList.add(MODEL_DIR + "/multi-model-one-ds.20.yaml");
+    createPushAuxiliaryImageWithDomainConfig(MII_AUXILIARY_IMAGE_NAME, miiAuxiliaryImage1Tag, archiveList, modelList);
+    // image2 with model files for jms config
+    modelList = new ArrayList<>();
+    modelList.add(MODEL_DIR + "/model.jms2.yaml");
+    WitParams witParams
+        = new WitParams()
+            .modelImageName(MII_AUXILIARY_IMAGE_NAME)
+            .modelImageTag(miiAuxiliaryImage2Tag)
+            .wdtModelOnly(true)
+            .modelFiles(modelList)
+            .wdtVersion("NONE");
+    createAndPushAuxiliaryImage(MII_AUXILIARY_IMAGE_NAME, miiAuxiliaryImage2Tag, witParams);
+
+    // admin/managed server name here should match with model yaml
+    String auxiliaryImagePath = "/auxiliary";
+    String clusterName = "cluster-1";
+
+    // create domain custom resource using 2 auxiliary images
+    logger.info("Creating domain custom resource with domainUid {0} and auxiliary images {1} {2}",
+        auxDomainUid, miiAuxiliaryImage1, miiAuxiliaryImage2);
+    DomainResource domainCR = CommonMiiTestUtils.createDomainResourceWithAuxiliaryImage(
```

Comment:
You can use static import here:
CommonMiiTestUtils.createDomainResourceWithAuxiliaryImage

Reply:
fixed